.github/workflows/tcpdump.yml

```
@@ -0,0 +1,126 @@
+        with:
+          repository: the-tcpdump-group/libpcap
+          path: libpcap_repo
+
+      - name: Build and install libpcap
+        working-directory: libpcap_repo
+        run: |
+          ./autogen.sh
+          ./configure --prefix=$GITHUB_WORKSPACE/libpcap-install
+          make -j$(nproc)
+          make install
+
+      - name: Checkout tcpdump
+        uses: actions/checkout@v4
+        with:
+          repository: the-tcpdump-group/tcpdump
+          path: tcpdump_repo
+          ref: ${{ matrix.tcpdump_ref }}
+
+      - name: Build and install tcpdump
+        working-directory: tcpdump_repo
+        run: |
+          ./autogen.sh
+          export PKG_CONFIG_PATH=$GITHUB_WORKSPACE/libpcap-install/lib/pkgconfig:$PKG_CONFIG_PATH
+          ./configure --prefix=$GITHUB_WORKSPACE/tcpdump-install --with-pcap=$GITHUB_WORKSPACE/libpcap-install
+          make -j$(nproc)
+          make install
+
+      - name: Run tcpdump tests
+        working-directory: tcpdump_repo
+        run: |
+          # Set environment variables for wolfProvider (for OpenSSL)
+          export LD_LIBRARY_PATH="$GITHUB_WORKSPACE/wolfssl-install/lib:$GITHUB_WORKSPACE/openssl-install/lib:$GITHUB_WORKSPACE/openssl-install/lib64:$GITHUB_WORKSPACE/libpcap-install/lib:$GITHUB_WORKSPACE/tcpdump-install/lib:$LD_LIBRARY_PATH"
+          export OPENSSL_CONF="$GITHUB_WORKSPACE/provider.conf"
+          export OPENSSL_MODULES="$GITHUB_WORKSPACE/wolfprov-install/lib"
+          export PKG_CONFIG_PATH="$GITHUB_WORKSPACE/openssl-install/lib64/pkgconfig:$GITHUB_WORKSPACE/libpcap-install/lib/pkgconfig:$PKG_CONFIG_PATH"
+          export PATH="$GITHUB_WORKSPACE/tcpdump-install/sbin:$GITHUB_WORKSPACE/tcpdump-install/bin:$PATH"
+          
+          # Apply the force_fail matrix variable
+          export ${{ matrix.force_fail }}
+
+          # Debug: Check if wolfProvider is loading correctly
+          echo "Testing wolfProvider availability:"
+          openssl list -providers || true
```

We need to check that our provider is listed in this command, and not mask any failure with `|| true`.

Something like this: 
```
$GITHUB_WORKSPACE/openssl-install/bin/openssl list -providers | grep libwolfprov
```